### PR TITLE
HOCS-5032: Update Secrets to match prod format

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: kube/kd/deployment.yaml [hunk 1]
```diff
@@ -205,33 +205,33 @@ spec:
...
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
-                  key: db_name
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: name
             - name: DB_SCHEMA_NAME
               valueFrom:
                 secretKeyRef:
                   name: hocs-info-service
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:223]: Does this need repointing?

File: kube/kd/deployment.yaml [hunk 2]
```diff
@@ -205,33 +205,33 @@ spec:
...
-                  key: rds_schema
+                  key: schema
             - name: DB_USERNAME
               valueFrom:
                 secretKeyRef:
                   name: hocs-info-service
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:228]: Does this need repointing?

File: kube/kd/deployment.yaml [hunk 3]
```diff
@@ -205,33 +205,33 @@ spec:
...
-                  key: rds_user
+                  key: user_name
             - name: DB_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: hocs-info-service
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:233]: Does this need repointing?

File: kube/kd/deployment.yaml [hunk 4]
```diff
@@ -70,18 +70,18 @@ spec:
...
-                  name: {{.KUBE_NAMESPACE}}-rds
+                  name: {{.KUBE_NAMESPACE}}-info-rds
                   key: jdbc_url
             - name: FLYWAY_USER
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_user
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: user_name
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:79]: This currently points to the root user? That expected?

File: kube/kd/deployment.yaml [hunk 5]
```diff
@@ -70,18 +70,18 @@ spec:
...
             - name: FLYWAY_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_password
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: password
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:84]: Same for this - root password?

File: kube/kd/deployment.yaml [hunk 6]
```diff
@@ -205,33 +205,33 @@ spec:
...
-                  name: {{.KUBE_NAMESPACE}}-rds
+                  name: {{.KUBE_NAMESPACE}}-info-rds
                   key: port
             - name: DB_NAME
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
-                  key: db_name
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: name
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:219]: `name` doesn't exist in prod

File: kube/kd/deployment.yaml [hunk 7]
```diff
@@ -205,33 +205,33 @@ spec:
...
             - name: DB_SCHEMA_NAME
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_schema
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: schema
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:224]: `schema` doesn't exist in prod

File: kube/kd/deployment.yaml [hunk 8]
```diff
@@ -205,33 +205,33 @@ spec:
...
             - name: DB_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_user
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: user_name
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:229]: This is `username` in prod and points to root, that what you expect?

File: kube/kd/deployment.yaml [hunk 9]
```diff
@@ -205,33 +205,33 @@ spec:
...
             - name: DB_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_password
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: password
```
Comment (LiamMacP) [inline kube/kd/deployment.yaml:234]: This points to `roots` password?

Comment (nickmcmahon01) [reply kube/kd/deployment.yaml:234]: hocs-info-service doesn't contain the root user, the cs-*-rds one does

File: kube/kd/deployment.yaml (hunk 7 above)
Comment (LiamMacP) [reply kube/kd/deployment.yaml:224]: This needs to be `schema_name`.

```suggestion
                  key: schema_name
```

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**